doc/manual/parallel-computing.rst

```diff
@@ -498,6 +561,66 @@ the backing store on the remote process.
 example of this is provided in ``examples/dictchannel.jl`` which uses a dictionary as its remote store.
 
 
+Channels and RemoteChannels
+---------------------------
+- A :class:`Channel` is local to a process.
```

Comment:
Maybe add a sentence saying something like "if we have two workers, worker 2 cannot interact with `Channel`s on worker 1"?